File: project2/Team1/spring/src/main/java/bitc/full502/spring/dto/LodgingDetailDto.java
package bitc.full502.spring.dto;

import lombok.*;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class LodgingDetailDto {
    // Lodging 기본정보
    private Long id;
    private String name;
    private String city;
    private String town;
    private String vill;
    private String phone;
    private String addrRd; // 도로명
    private String addrJb; // 지번
    private Double lat; // 위도
    private Double lon; // 경도
    private Integer totalRoom;
    private String img;

    // 집계 정보
    private Long views;      // 조회수
    private Long wishCount;  // 찜 수
    private Long bookCount;  // 예약 수(취소 제외, 임시 규칙)
}

File: project2/Team1/android/app/src/main/java/bitc/fullstack502/android_studio/network/dto/LodgingDetailDto.kt
package bitc.fullstack502.android_studio.network.dto

data class LodgingDetailDto(
    val id: Long,
    val name: String?,
    val city: String?,
    val town: String?,
    val vill: String?,
    val phone: String?,
    val addrRd: String?,
    val addrJb: String?,
    val lat: Double?,
    val lon: Double?,
    val totalRoom: Int?,
    val img: String?,
    val views: Long?,
    val wishCount: Long?,
    val bookCount: Long?
)
